example-conf/mesos-dns/mesos-site.xml

```diff
@@ -43,7 +43,7 @@
 
   <property>
     <name>mesos.hdfs.zkfc.ha.zookeeper.quorum</name>
-    <value>master0.mesos:2181,master1.mesos:2181,master2.mesos:2181</value>
```

Comment:
But when we failover, our ZKFC's have to restart and reconnect to a new Zookeeper and state is lost. Ideally, we would have 3 to begin with. Is there an issue with having 3 as a default?